x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.ts

```diff
@@ -179,50 +179,261 @@ export function getSpacesUsageCollector(
+          },
+        },
+        dashboard: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        ml: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+        apm: {
+          type: 'long',
+          _meta: {
+            description: 'The number of spaces which have this feature disabled.',
+          },
+        },
+      },
+      available: {
+        type: 'boolean',
+        _meta: {
+          description: 'Indicates if the spaces feature is available in this installation.',
+        },
+      },
+      enabled: {
+        type: 'boolean',
+        _meta: {
+          description: 'Indicates if the spaces feature is enabled in this installation.',
+        },
+      },
+      count: {
+        type: 'long',
+        _meta: {
+          description: 'The number of spaces in this installation.',
+        },
+      },
+      'apiCalls.copySavedObjects.total': {
+        type: 'long',
+        _meta: {
+          description: 'The number of times the "Copy to Space" API has been called.',
+        },
+      },
+      'apiCalls.copySavedObjects.kibanaRequest.yes': {
+        type: 'long',
+        _meta: {
+          description: 'The number of times the "Copy to Space" API has been called via Kibana.',
+        },
+      },
+      'apiCalls.copySavedObjects.kibanaRequest.no': {
+        type: 'long',
+        _meta: {
+          description:
+            'The number of times the "Copy to Space" API has been called via an API consumer (e.g. curl).',
+        },
+      },
+      'apiCalls.copySavedObjects.createNewCopiesEnabled.yes': {
+        type: 'long',
+        _meta: {
+          description:
+            'The number of times the "Copy to Space" API has been called with "createNewCopies" set to true.',
+        },
+      },
+      'apiCalls.copySavedObjects.createNewCopiesEnabled.no': {
+        type: 'long',
+        _meta: {
+          description:
+            'The number of times the "Copy to Space" API has been called with "createNewCopies" set to false.',
+        },
+      },
+      'apiCalls.copySavedObjects.overwriteEnabled.yes': {
+        type: 'long',
+        _meta: {
+          description:
+            'The number of times the "Copy to Space" API has been called with "overwrite" set to true.',
+        },
+      },
+      'apiCalls.copySavedObjects.overwriteEnabled.no': {
+        type: 'long',
+        _meta: {
+          description:
+            'The number of times the "Copy to Space" API has been called with "overwrite" set to false.',
+        },
+      },
+      'apiCalls.resolveCopySavedObjectsErrors.total': {
+        type: 'long',
+        _meta: {
+          description:
+            'The number of times the "Resolve Copy to Space Conflicts" API has been called.',
```

Comment:
Same sentiment as my earlier comment... the HTTP route is `_resolve_copy_saved_objects_errors`, how do you feel about changing this to match?